File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int i = 1;
        while (true) {
            int n = sc.nextInt();

            if (n == 0) {
                return;
            }

            if (n % 2 == 0) {
                System.out.println(i + ". even " + n / 2);
            } else {
                System.out.println(i + ". odd " + ((n-1) / 2));
            }

            i++;
        }
    }
}

File: 백준/Bronze/4892. 숫자 맞추기 게임/숫자 맞추기 게임.cc
#include <iostream>
using namespace std;

int main() {

	int i = 1;
	while (true)
	{
		int n;
		cin >> n;

		if (n == 0)
		{
			return 0;
		}


		if (n % 2 != 0)
		{
			cout << i << ". odd " << (n - 1) / 2 << endl;
		}
		else {
			cout << i << ". even " << (n / 2) << endl;
		}

		i++;
	}
}